apps/frontend/nuxt.config.ts

```diff
@@ -8,7 +8,7 @@ import { globIterate } from "glob";
 import { match as matchLocale } from "@formatjs/intl-localematcher";
 import { consola } from "consola";
 
-const STAGING_API_URL = "https://staging-api.modrinth.com/v2/";
+const STAGING_API_URL = "https://api.modrinth.com/v2/";
```

Comment:
This will revert before merge.

Reply:
You can just create `.env` file in the Knossos directory and override `API_BASE_URL` (I think that's the var?).